tests/test_sqlcompletion.py

```diff
@@ -4,10 +4,17 @@
 from pgcli.packages.parseutils.tables import TableReference
 import pytest
 
-# Returns the expected select-clause suggestions for a single-table select
+
+def table_reference(table, schema=None, alias=None, is_function=False):
```

Comment:
On reflection, probably none, I think I added it before I did a bunch of line-wrapping. Of course, one can avoid passing a few parameters since `table_reference` has default parameters, but I don't think the method pulls its weight, I'll remove it.